packages/lesswrong/lib/notificationTypes.tsx

```diff
@@ -29,7 +29,7 @@ import DebateIcon from '@material-ui/icons/Forum';
 import DialogueChecks from './collections/dialogueChecks/collection';
 import { taggingNamePluralSetting } from './instanceSettings';
 
-export const notificationDocumentTypes = new TupleSet(['post', 'comment', 'user', 'message', 'tagRel', 'localgroup', 'dialogueCheck'] as const)
+export const notificationDocumentTypes = new TupleSet(['post', 'comment', 'user', 'message', 'tagRel', 'localgroup', 'dialogueCheck', 'newDialogueChecks'] as const)
```

Comment:
I don't think this needs a new document type

Reply:
Agreed, it should be `null`